x/concentrated-liquidity/tick_test.go

```diff
@@ -121,3 +121,207 @@ func (s *KeeperTestSuite) TestNextInitializedTick() {
 		})
 	})
 }
+
+func (s *KeeperTestSuite) TestInitOrUpdateTick() {
+	const validPoolId = 1
+	type param struct {
+		poolId      uint64
+		tickIndex   int64
+		liquidityIn sdk.Dec
+		upper       bool
+	}
+
+	tests := []struct {
+		name                   string
+		param                  param
+		tickExists             bool
+		expectedLiquidityNet   sdk.Dec
+		expectedLiquidityGross sdk.Dec
+		expectedErr            string
```

Comment:
Can we have error here instead of string?

If someone changes an error message, all tests have to be updated. On the other hand, if we expect an error here, the additional changes wouldn't be necessary